flask_server.py

```diff
@@ -311,13 +296,14 @@ def root():
     return redirect("/site-map", code=302)
 
 
-if __name__ == "__main__":
+def run_flask(default_host: str = "0.0.0.0", default_port: int = 8503) -> None:
```

Comment:
I moved the default port to the method call for now. @fortrieb 

If don't know if the flask setup should be put in a config json here. This PR focuses on introducing ruff rules. And the rule I fixed here was "no magic number".